### PR TITLE
Feat 3081 - create crm (former admin panel) app

### PR DESCRIPTION
**Pull request by phpgeekfr**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -1193,6 +1193,137 @@
...
+            ],
+            "styles": [
+              "node_modules/angular-calendar/css/angular-calendar.css",
+              "apps/crm/crm/src/styles.scss"
+            ],
+            "scripts": []
+          },
+          "configurations": {
+            "production": {
+              "fileReplacements": [
```
Comment (GrandSchtroumpf) [inline angular.json:1237]: check how it's done with the other app, but I think you'll need to update that.

File: angular.json [hunk 2]
```diff
@@ -1193,6 +1193,137 @@
...
+              "apps/crm/crm/src/favicon.ico",
+              "apps/crm/crm/src/assets",
+              {
+                "glob": "**/*",
+                "input": "./libs/ui/src/lib/assets",
+                "output": "./assets"
+              }
+            ],
+            "styles": [
+              "node_modules/angular-calendar/css/angular-calendar.css",
```
Comment (GrandSchtroumpf) [inline angular.json:1230]: Do you need the calendar ?

File: angular.json [hunk 3]
```diff
@@ -1193,6 +1193,137 @@
...
+              "apps/crm/crm/src/styles.scss"
+            ],
+            "scripts": []
```
Comment (GrandSchtroumpf) [inline angular.json:1233]: You might need to add `@firebase` in somewhere here. Something like `externalDependancies`

File: apps/crm/crm/src/app/app.module.ts [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+import { MatNativeDateModule } from '@angular/material/core';
+
+
+@NgModule({
+  declarations: [AppComponent],
+  imports: [
+    
+    // Angular
+    BrowserModule.withServerTransition({ appId: 'crm-app' }),
```
Comment (GrandSchtroumpf) [inline apps/crm/crm/src/app/app.module.ts:41]: Did you created a ssr or prerendering ?

File: apps/crm/crm/src/styles.scss [hunk 5]
```diff
@@ -0,0 +1,36 @@
+/* You can add global styles to this file, and also import other style files */
+@import "../../../../styles/style.scss";
+
+.light-theme {
+  @include angular-material-theme($bf-light-theme);
+  @include sat-datepicker-theme($bf-light-theme);
```
Comment (GrandSchtroumpf) [inline apps/crm/crm/src/styles.scss:6]: You can remove that

File: apps/crm/crm/src/styles.scss [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  @include bf-material-theme($bf-light-theme);
+  @include bf-calendar-theme($bf-light-theme);
```
Comment (GrandSchtroumpf) [inline apps/crm/crm/src/styles.scss:8]: Same here I think.

File: angular.json (hunk 3 above)
Comment (phpgeekfr) [reply angular.json:1233]: It seams that this is only used with festival "server".

**Pull request merged**